test/blackbox-tests/melange-playground/compile-ml-warning-alert.t

```diff
@@ -2,6 +2,7 @@
   > require(process.env.DUNE_SOURCEROOT + '/_build/default/bin/jsoo_main.bc.js');
   > require(process.env.DUNE_SOURCEROOT + '/_build/default/bin/melange-cmijs.js');
   > console.log(ocaml.compileML("let t = Js.Vector.length"));
+  > console.log(ocaml.compileRE('external foo: (([@mel.uncurry] (unit => unit)) => [@mel.uncurry] (unit => unit)) => unit = "Array.from"'));
```

Comment:
Could we add this in a new test `compile-re-warning-alert.t` pls?

Besides being able to parallelize more, I have been trying to just include a single compilation on each test because  otherwise it gets hard to track down which part of the output belongs to which input.

Reply:
absolutely!